game/src/edit/traffic_signals/offsets.rs

```diff
@@ -68,22 +67,10 @@ impl State<App> for ShowAbsolute {
         ctx.canvas_movement();
         if ctx.redo_mouseover() {
             app.primary.current_selection = app.mouseover_unzoomed_roads_and_intersections(ctx);
+            app.select_only_intersections(|i| self.members.contains(&i));
```

Comment:
The changes in this file are the best demonstration of the new App helpers. After auditing lots more code, they might also be some of the only places really benefitting from it. :\